### DISCUSSION TITLE
Fleet API

### DISCUSSION THREAD

**Discussion by brian-gates** [alandtse/tesla#742] [General]
Thoughts about Tesla's just-announced [Fleet API](https://developer.tesla.com/docs/fleet-api)? Could it be leveraged in this library?

**Comment by joperezr** [comment]
Looks like the new API is a Go SDK, so unless I'm mistaken,  a python binding would need to be created on top of the Go SDK so that this integration can continue to work. I created [this issue](https://github.com/zabuldon/teslajsonpy/issues/439) in teslajsonpy repo to see if they can consider adding support for the new SDK, but it sounds like it would actually be hard to accomplish this.
